db/config.cc

```diff
@@ -1327,6 +1327,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , maximum_replication_factor_fail_threshold(this, "maximum_replication_factor_fail_threshold", liveness::LiveUpdate, value_status::Used, -1, "")
     , tablets_initial_scale_factor(this, "tablets_initial_scale_factor", value_status::Used, 10,
          "Minimum average number of tablet replicas per shard per table. Suppressed by tablet hints in table's schema: min_per_shard_tablet_count and min_tablet_count")
+    , tablets_per_shard_goal(this, "tablets_per_shard_goal", liveness::LiveUpdate, value_status::Used, 100,
+         "The goal for the maximum number of tablet replicas per shard. Tablet allocator tries to keep this goal.")
```

Comment:
nit in commit tablets, topology: Introduce tablets_per_shard_goal config log message: s/decrese/decrease/, s/similat/similar/